plinth/modules/filetea/__init__.py

```diff
@@ -0,0 +1,148 @@
+#
+# This file is part of Plinth.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+"""
+Plinth module to configure FileTea.
+"""
+
+from django.utils.translation import ugettext_lazy as _
+
+from plinth import actions
+from plinth import action_utils
+from plinth import cfg
+from plinth import frontpage
+from plinth import service as service_module
+from plinth.menu import main_menu
+from plinth.utils import format_lazy
+from plinth.modules.config.config import get_domainname
+
+version = 1
+
+managed_services = ['filetea']
+
+managed_packages = ['filetea']
+
+title = _('File Sharing \n (FileTea)')
+
+description = [
+    _('FileTea is an anonymous, volatile file sharing service. '
+      'Volatile means the shared file is available only as long '
+      'as the sender keeps FileTea open on their browser.'),
+    format_lazy(
+        _('Running FileTea on {box_name} allows you to share files securely '
+          'with other users of {box_name}. On a public FileTea service, '
+          'anyone with the link can download a shared file. But FileTea on '
+          'FreedomBox is password protected, so only registered users of that '
+          'FreedomBox can download shared files, not any snooping third '
+          'party.'),
+        box_name=_(cfg.box_name)),
+    _('When enabled, FileTea\'s web interface will be available from '
+      '<a href=\"https://{}:8686\">filetea</a>.'.format(get_domainname())),
+]
+
+
+def init():
+    """Intialize the module."""
+    menu = main_menu.get('apps')
+    menu.add_urlname(title, 'glyphicon-share', 'filetea:index')
+
+    global service
+    setup_helper = globals()['setup_helper']
+    if setup_helper.get_state() != 'needs-setup':
+        service = service_module.Service(
+            managed_services[0],
+            title,
+            ports=['http', 'https'],
```

Comment:
I think you will want to replace the ports here, with the service name "filetea-plinth".